configure.ac

```diff
@@ -121,6 +121,8 @@ AC_ARG_ENABLE(test-Werror,
   AS_IF([test x$enable_test_werror != xno], [
       TESTSUITE_CFLAGS="${TESTSUITE_CFLAGS} -Werror -W -Wpointer-arith"
   ])
+AX_CHECK_COMPILE_FLAG([-Wno-format],
+    [SKELETONS_CFLAGS="$SKELETONS_CFLAGS -Wno-format"])
```

Comment:
The formats should work without warnings. What kind of warnings do you see that require this flag? Do you refer to `%zd/%zu` in -std=c89 mode?

Reply:
Yes, exactly. Here is the travis log
https://travis-ci.org/velichkov/asn1c/jobs/289572223#L1129-L1137

Reply:
OK.